backend/src/api/v1/dto/CategoryDTO.ts

```diff
@@ -0,0 +1,42 @@
+import BaseDTO from './BaseDTO';
+import PlaylistDTO from './PlaylistDTO';
+
+// https://github.com/bennyn/wlc-webapp/blob/wlc-webapp/0.0.36/src/
```

Comment:
Breaking a link into multiple lines isn't very practical because you cannot double click it to view it.